lib/generators/factory_bot/authentication/authentication_generator.rb

```diff
@@ -0,0 +1,12 @@
+require "generators/factory_bot"
+require "factory_bot_rails"
+
+module FactoryBot
+  module Generators
+    class AuthenticationGenerator < Base
+      def create_fixture_file
+        template 'users.rb', 'test/factories/users.rb'
```

Comment:
I don't think `test/factories/users.rb` should be static here. It doesn't allow overriding things like the factory_bot directory, suffix, etc.

I think you'll need to [include options](https://github.com/thoughtbot/factory_bot_rails/blob/main/lib/generators/factory_bot/model/model_generator.rb#L14-L26), and [build up the name](https://github.com/thoughtbot/factory_bot_rails/blob/main/lib/generators/factory_bot/model/model_generator.rb#L70-L81) [dynamically](https://github.com/thoughtbot/factory_bot_rails/blob/main/lib/generators/factory_bot/model/model_generator.rb#L50-L53).

Reply:
Sounds good, maybe I can extract those common functions into a module